### PR TITLE
Update map tool notes to indicate compliance with J2735 2024 version

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
The team recently compared the 2016 and 2024 versions of the MAP message in J2735 and found no differences for the MAP definition.  This story will update the notes for MAP Tool to indicate compliance with the 2024 version.  

Update version number on home page (index.html).
Update version number on ISD create page and TIM create page 
## Related GitHub Issue
The current MapTool notes still indicate it is using J2735 2016 version.

## Related Jira Key
https://usdot-carma.atlassian.net/browse/MAP-127

## Motivation and Context
NA

## How Has This Been Tested?
NA

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: fedgov-cv-ISDcreator-webapp/src/main/webapp/index.html [hunk 1]
```diff
@@ -117,7 +117,7 @@
 							<li><a href="#instructions_modal" data-toggle="modal" data-target="#instructions_modal">Instructions</a></li>
 							<li class="divider"></li>
 							<li><a href="#" onclick="alert('To request support or report problems please email\nthe CAV Support Services at: CAVSupportServices@dot.gov.');">Support</a></li>
-							<li><a href="#" data-toggle="modal" data-target="#" onclick="alert('ISD Builder Tool for J2735 3/2016 (UPER encoding)\n\n\t  Version 2.2.8, Released on 5/2/2019');">About</a></li>
+							<li><a href="#" data-toggle="modal" data-target="#" onclick="alert('ISD Builder Tool for J2735 9/2024 (UPER encoding)\n\n\t  Version 2.2.8, Released on 5/2/2019');">About</a></li>
```
Comment (DokurOmkar) [inline fedgov-cv-ISDcreator-webapp/src/main/webapp/index.html:120]: I believe we may have to update the line "Version 2.2.8, Released on 5/2/2019" to the new release version and release date.

File: fedgov-cv-TIMcreator-webapp/src/main/webapp/index.html [hunk 2]
```diff
@@ -101,7 +101,7 @@
 							<li><a href="#instructions_modal" data-toggle="modal" data-target="#instructions_modal">Instructions</a></li>
 							<li class="divider"></li>
 							<li><a href="#" onclick="alert('To request support or report problems please email\nthe CAV Support Services at: CAVSupportServices@dot.gov.');">Support</a></li>
-							<li><a href="#" data-toggle="modal" data-target="#" onclick="alert('TIM Builder Tool for J2735 3/2016 (UPER encoding)\n\n\t  Version 2.2.4, Released on 3/26/2019');">About</a></li>
+							<li><a href="#" data-toggle="modal" data-target="#" onclick="alert('TIM Builder Tool for J2735 9/2024 (UPER encoding)\n\n\t  Version 2.2.4, Released on 3/26/2019');">About</a></li>
```
Comment (DokurOmkar) [inline fedgov-cv-TIMcreator-webapp/src/main/webapp/index.html:104]: Same as above

**Review by DokurOmkar — CHANGES_REQUESTED** [review]

File: fedgov-cv-ISDcreator-webapp/src/main/webapp/index.html (hunk 1 above)
Comment (dan-du-car) [reply fedgov-cv-ISDcreator-webapp/src/main/webapp/index.html:120]: @DokurOmkar What is the new release version and date?

File: fedgov-cv-TIMcreator-webapp/src/main/webapp/index.html (hunk 2 above)
Comment (dan-du-car) [reply fedgov-cv-TIMcreator-webapp/src/main/webapp/index.html]: https://github.com/usdot-fhwa-stol/connectedvcs-tools/pull/34/commits/b25bfdd1c55ac10055be9e009675692df90e732b

**Review by DokurOmkar — APPROVED** [review]

**Pull request merged**